core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceFactoryImpl.java

```diff
@@ -61,14 +68,16 @@
 	@Reference
 	CryptoSupport cryptoSupport;
 
+  @Reference
+  private ResourceResolverFactory resolverFactory;
+
```

Comment:
It looks like the bootstrap code is using a different indentation than your editor is.. you should probably resolve this one way or the other